### ISSUE TITLE
unrecognized field "most_recent_feature_version" 

### ISSUE THREAD

**Issue by maxandersen** [AdoptOpenJDK/openjdk-api-java-client#6]
```
//usr/bin/env jbang "$0" "$@" ; exit $?
//DEPS net.adoptopenjdk:net.adoptopenjdk.v3.api:0.3.3
//DEPS net.adoptopenjdk:net.adoptopenjdk.v3.vanilla:0.3.3

import net.adoptopenjdk.v3.api.AOV3Exception;
import net.adoptopenjdk.v3.vanilla.AOV3Clients;

import java.io.IOException;

class adoptopenjdk {

    public static void main(String... args) throws Exception {
        var clients = new AOV3Clients();
        try (var client = clients.createClient()) {
            var request = client.availableReleases(e -> System.out.println(e));
            var releases = request.execute();
        }
    }
}
```

results in:
`Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "most_recent_feature_version" (class net.adoptopenjdk.v3.vanilla.internal.AOV3AST$AOV3AvailableReleasesJSON), not marked as ignorable (4 known properties: "available_releases", "most_recent_lts", "available_lts_releases", "most_recent_feature_release"])
`

**Comment by maxandersen** [comment]
same error happens when running tests:

```
[INFO] Running net.adoptopenjdk.v3.tests.AOV3ClientsIntegrationTest
[ERROR] Tests run: 10, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 5.242 s <<< FAILURE! - in net.adoptopenjdk.v3.tests.AOV3ClientsIntegrationTest
[ERROR] net.adoptopenjdk.v3.tests.AOV3ClientsIntegrationTest.testAvailableReleases  Time elapsed: 0.491 s  <<< ERROR!
net.adoptopenjdk.v3.api.AOV3ExceptionParseFailed:
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "most_recent_feature_version" (class net.adoptopenjdk.v3.vanilla.internal.AOV3AST$AOV3AvailableReleasesJSON), not marked as ignorable (4 known properties: "available_releases", "most_recent_lts", "available_lts_releases", "most_recent_feature_release"])
 at [Source: (GZIPInputStream); line: 17, column: 38] (through reference chain: net.adoptopenjdk.v3.vanilla.internal.AOV3AST$AOV3AvailableReleasesJSON["most_recent_feature_version"])
	at net.adoptopenjdk.v3.tests.AOV3ClientsIntegrationTest.testAvailableReleases(AOV3ClientsIntegrationTest.java:66)
Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException:
Unrecognized field "most_recent_feature_version" (class net.adoptopenjdk.v3.vanilla.internal.AOV3AST$AOV3AvailableReleasesJSON), not marked as ignorable (4 known properties: "available_releases", "most_recent_lts", "available_lts_releases", "most_recent_feature_release"])
 at [Source: (GZIPInputStream); line: 17, column: 38] (through reference chain: net.adoptopenjdk.v3.vanilla.internal.AOV3AST$AOV3AvailableReleasesJSON["most_recent_feature_version"])
	at net.adoptopenjdk.v3.tests.AOV3ClientsIntegrationTest.testAvailableReleases(AOV3ClientsIntegrationTest.java:66)
```

**Comment by joschi** [comment]
Refs https://github.com/AdoptOpenJDK/openjdk-api-v3/issues/217
Refs https://github.com/AdoptOpenJDK/openjdk-api-v3/pull/218

**Comment by io7m** [comment]
Fixed in #7, and insulated against future changes in #8.
